.github/workflows/component-builder.yml

```
@@ -23,40 +28,53 @@ jobs:
             libvirt-daemon-system \
             virtinst cloud-image-utils \
             libguestfs-tools
+          case "${{ matrix.CPU_ARCH }}" in
+            x86_64)
+              sudo apt-get install -y qemu-system-x86
+              ;;
+            aarch64)
+              sudo apt-get install -y qemu-system-arm qemu-efi-aarch64
+              ;;
+            s390x)
+              sudo apt-get install -y qemu-system-s390x 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)